certora/harness/ReserveConfigurationHarness.sol

```diff
@@ -1,20 +1,22 @@
 pragma solidity 0.8.10;
 pragma experimental ABIEncoderV2;
 
-import {ReserveConfiguration} from '../munged/protocol/libraries/configuration/ReserveConfiguration.sol';
+//import {ReserveConfiguration} from '../munged/protocol/libraries/configuration/ReserveConfiguration.sol';
+import {ReserveConfiguration} from './ReserveConfiguration_noBV.sol';
 import {DataTypes} from '../munged/protocol/libraries/types/DataTypes.sol';
+import {PoolStorage} from './PoolStorage_noBV.sol';
 
-contract ReserveConfigurationHarness {
+contract ReserveConfigurationHarness is PoolStorage {
```

Comment:
Why do you need it to be inherited?

Reply:
I tried that to mitigate some issues, but it's not gonna be necessary.